narayana/ArjunaJTA/jta/tests/classes/com/arjuna/ats/jta/xa/performance/HQStoreBenchmark.java

```diff
@@ -16,20 +16,24 @@
  */
 package com.arjuna.ats.jta.xa.performance;
 
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.openjdk.jmh.annotations.Benchmark;
+import org.openjdk.jmh.annotations.Level;
+import org.openjdk.jmh.annotations.Scope;
+import org.openjdk.jmh.annotations.Setup;
+import org.openjdk.jmh.annotations.State;
+import org.openjdk.jmh.infra.Blackhole;
+
```

Comment:
These changes are code clean up. Normally we only allow PR changes that either:

- specifically address the issue being fixed, or
- if the changes are clean up then we use a separate git commit that is included with the PR

So this change, although a good change, should be performed in [issue JBTM-3247](https://issues.redhat.com/browse/JBTM-3247)  or in a separate commit as part of this PR.

I have added this procedure to our Narayana New Hire Orientation document. I guess they should be in some general policy document for the broader community but we do not have one yet.

Reply:
Okay, I will undo these changes from this pull request and, will do these changes as part of  issue JBTM-3247